src/widgets/index.js

```diff
@@ -28,6 +30,6 @@ var widgets = Object.assign(
 )
 
 module.exports = widgets
-console.log('@@@ widgets: ', widgets)
+debug.log('@@@ widgets: ', widgets)
```

Comment:
We need to check with Tim how valuable this line actually is currently. It's really annoying in tests.

Reply:
Agree! :)